dist/stat_data.py

```diff
@@ -275,6 +275,7 @@ def __init__(self, name, desc, flags=''):
     CacheStat('cache_eviction_force_clean', 'forced eviction - pages evicted that were clean count'),
     CacheStat('cache_eviction_force_clean_time', 'forced eviction - pages evicted that were clean time (usecs)'),
     CacheStat('cache_eviction_force_delete', 'forced eviction - pages selected because of too many deleted items count'),
```

Comment:
Can we change the description of this stat, as now it has a different meaning?

`forced eviction - pages selected because of all items found deleted`

or somewhere along these lines.

Reply:
Thanks. Fixed as `forced eviction - pages selected because of all deleted items found`